recipes/hunspell-en/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: hunspell-en
+  version: 2014.02.10
+
+source:
+  url: http://extensions.openoffice.org/en/download/17707
```

Comment:
Apparently this always points to the latest version, which has already been updated.  Does anyone know how to navigate sourceforge? https://sourceforge.net/projects/aoo-extensions/